lib/ios/RNNRootViewController.m

```diff
@@ -43,8 +44,10 @@ -(instancetype)initWithName:(NSString*)name
 
 -(void)viewWillAppear:(BOOL)animated{
 	[super viewWillAppear:animated];
-	[self.options applyOn:self];
-	[self optionsUpdated];
+	if (!self._optionsApplied) {
+		[self.options applyOn:self];
+	}
+	self._optionsApplied = true;
```

Comment:
Is there a case where you need to reset this to `false`, say after `viewDidDisappear()` or something?

Reply:
don't think so. wix mentioned maybe to do this on `viewDidLoad` but some of the data points aren't available.. they accepted this pr. https://github.com/wix/react-native-navigation/pull/2982